examples/example_app/example_app/templates/example_app/custom_job_form.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'extras/job.html' %}
+{% load form_helpers %}
+<!-- The `job_form` block is where the actual job form is displayed. -->
+{% block job_form %}
+
+<!-- 
+    We are going to render a simple HTML form element with bootstrap formatting.
+    The field name is `custom_job_data` which exactly matches the script variable name
+    in the job.
+
+    Note that we are not calling `block.super` here because we don't want any
+    part of the parent job_form block to be rendered (in this case).
+-->
+<div class="form-group">
+    {% with field=job_form.fields.custom_job_data %}
+        {% with field_value=job_form.initial.custom_job_data|default:field.initial %}
+    <label class="col-md-3 control-label required" for="id_custom_job_data">{{ field.label }}</label>
+    <div class="col-md-9">
+        <textarea id="id_custom_job_data" name="custom_job_data">{{ field_value }}</textarea>
```

Comment:
Might be that adding `class="form-control"` to the textarea might improve its rendering - in the provided screenshot it doesn't look like it's styled consistently with the rest of the form at present.

Reply:
Done.